python_scripts/03_categorical_pipeline.py

```diff
@@ -276,30 +278,37 @@
 data["native-country"].value_counts()
 
 # %% [markdown]
-# We see that the `Holand-Netherlands` category is occurring rarely. This will
+# We see that the `"Holand-Netherlands"` category is occurring rarely. This will
 # be a problem during cross-validation: if the sample ends up in the test set
 # during splitting then the classifier would not have seen the category during
 # training and will not be able to encode it.
 #
-# In scikit-learn, there are two solutions to bypass this issue:
+# In scikit-learn, there are some possible solutions to bypass this issue:
 #
-# * list all the possible categories and provide it to the encoder via the
-#   keyword argument `categories`;
-# * use the parameter `handle_unknown`, i.e. if an unknown category is
+# * list all the possible categories and provide them to the encoder via the
+#   keyword argument `categories` instead of letting the estimator to
+#   automatically determine them from the training data when calling fit;
+# * set the parameter `handle_unknown="ignore"`, i.e. if an unknown category is
 #   encountered during transform, the resulting one-hot encoded columns for this
-#   feature will be all zeros.
+#   feature will be all zeros;
+# * adjust the `min_frequency` parameter to collapse the rarest categories
+#   observed in the training data into a single one-hot encoded feature. If you
+#   enable this option, you can also set `handle_unknown="infrequent_if_exist"`
+#   to encode the unknown categories (categories only observed at predict time)
+#   as ones in that last column.
 #
-# Here, we will use the latter solution for simplicity.
+# Here we demo the second option, but feel free to evaluate the alternatives.
```

Comment:
I find that "here" followed by the note about `OrdinalEncoder` breaks makes it hard to follow the though process.

```suggestion
# In this notebook we only explore the second option,
# `OneHotEncoder(handle_unknown="ignore")`, but feel free to evaluate the
# alternatives on your own, for instance using a sandbox notebook.
```